pkg/store/s3.go

```diff
@@ -110,12 +112,22 @@ func DownloadFromS3(cred S3Credentials, src string, dest string) (string, error)
 		return "", trace.Wrap(err)
 	}
 
+	s3Object, err := client.GetObject(loc.Bucket, loc.Path)
```

Comment:
sure, why not. It's more high level than GetObject, it wraps it and add additional checks for local FS, so use that